src/components/BodySection.js

```diff
@@ -524,8 +529,13 @@ export default function BodySection(props) {
 
         const JSX = (
           <div>
-            <p>{inputId}</p>
+            <p className="font-bold">{inputId}</p>
             <ol className="list-decimal">{listItems}</ol>
+            {sum >= 9 && (
+              <p className="font-bold">
+                Limit exercises for this muscle

```

Comment:
Maybe a text could always show here and it is either "No strain", "Low strain", "Medium Strain" "High strain", "ABORT YOU PSYCHO" and the text changes color depending on how many exercises for the muscle is selected.

Or just change this one to "Reduce nr of exercises for this muscle". 
When I saw "Limit" I thought it would be like something I could interact with somehow.

Reply:
I've updated this now to be "Low Strain", "Medium Strain", "High Strain" and "At Risk of Overtraining".

Reply:
Brilliant!